### PR TITLE
JS docs review

### PR DESCRIPTION
**Pull request by fliepeltje**
### Summary of changes

I've gone through all the js docs to review them; I've made some very minor changes:
- fixed some typos
- added objectives for all frameworks
- made some things a bit more consistent

### CODE REVIEW

**Review by mentels — COMMENTED** [review]
I see room for improvement.

But I think it's on you how far you want to take it.

Thanks, that is a great job anyway!

File: js/frameworks/svelte.html.markerb [hunk 1]
```diff
@@ -157,15 +158,13 @@ The rest of the file contains settings to be applied to the application when it
 
 ## Deploying to Fly.io
 
-We are now ready to deploy our containerized app to Fly.io. At the command line, just run:
+We are now ready to deploy our containerized app to Fly.io:
```
[the remark this answers is not in the document]
Comment (fliepeltje) [reply js/frameworks/svelte.html.markerb:161]: I'm unsure; I tried to not mess with the general content too much :sweat_smile: 

You could argue that you have a containerized app locally and deploy that; what happens on the server is a black box :P

File: js/frameworks/remix.html.markerb [hunk 2]
```diff
@@ -5,6 +5,7 @@ sitemap: false
 redirect_from:
  - /docs/languages-and-frameworks/remix/
  - /docs/getting-started/remix/
+objective: Remix is a full stack web framework that lets you focus on the user interface and work back through web standards to deliver a fast, slick, and resilient user experience.
```
[the remark this answers is not in the document]
Comment (fliepeltje) [reply js/frameworks/remix.html.markerb:8]: `ctrl-c` `ctrl-v` :v: :relieved:

File: js/frameworks/nuxtjs.html.markerb [hunk 3]
```diff
@@ -5,20 +5,25 @@ sitemap: false
...
  - /docs/getting-started/nuxtjs/
+objective: Nuxt is the intuitive Vue Framework.
 order: 6
 ---
 
 <% app_name = "hello-nuxt" %>
 <%= partial "partials/intro", locals: { runtime: "Nuxt", link: "https://nuxt.com" } %>
 
-You can deploy your [Nuxt](https://nuxt.com) app on Fly with minimal effort, our CLI will do the heavy lifting. You can use your existing Nuxt app or you can [create one using the tutorial](https://nuxt.com/docs/getting-started/installation) then come back here to deploy your app.
+You can deploy your [Nuxt](https://nuxt.com) app on Fly with minimal effort, our CLI will do the heavy lifting. You can use your existing Nuxt app or you can [create one using the tutorial](https://nuxt.com/docs/getting-started/installation).
```
[the remark this answers is not in the document]
Comment (fliepeltje) [reply js/frameworks/nuxtjs.html.markerb:15]: hmmmm, but there is a section for something tutorial-like: https://fly.io/docs/js/frameworks/nextjs/ :grimacing: 

Maybe we can discuss with @anniebabannie about what makes sense here and reconcile this in a separate PR.

[the remark this answers is not in the document]
Comment (mentels) [reply js/frameworks/nuxtjs.html.markerb:15]: I mean in 

> You can deploy your [Next.js](https://nextjs.org/) app on Fly with minimal effort, our CLI will do the heavy lifting. We’ll be using a standard app generated with [create-next-app](https://nextjs.org/docs/pages/api-reference/create-next-app) CLI tool provided by Next.js.

We're not sending them to a different page to build their app.

**Review by mentels — APPROVED** [review]

File: js/frameworks/nuxtjs.html.markerb (hunk 3 above)
[the remark this answers is not in the document]
Comment (fliepeltje) [reply js/frameworks/nuxtjs.html.markerb]: I've removed all tutorial references in 6746e14 - we can always go back if someone thinks it's a bad idea :)

File: js/frameworks/redwood.html.markerb [hunk 4]
```diff
@@ -13,18 +14,18 @@ order: 10
...
 
 ```cmd
 yarn create redwood-app my-redwood-project
 cd my-redwood-project
 ```
 
-Now check out the [Redwood Quick Start guide](https://redwoodjs.com/docs/quick-start) to get your development
-environment setup and to generate some data models.
+You can check out the [Redwood Quick Start guide](https://redwoodjs.com/docs/quick-start) to get your development
+environment setup and to generate some data models. Once you're happy with your app, we can deploy it:
```
[the remark this answers is not in the document]
Comment (fliepeltje) [reply js/frameworks/redwood.html.markerb:29]: Also resolved in 6746e14

**Review by anniebabannie — APPROVED** [review]
All looks good! Thanks for making these tweaks

**Pull request merged**